elastic/kibana
# [Synthetics] do not filter last check result by timespan

## Summary
Resolves https://github.com/elastic/kibana/issues/142657

Ensures that last status by location metric does not include a filter for `monitor.timespan`. 

When the monitor becomes off-schedule for any reason, such as failing to sync with the service or a disruption in communication between the service and ES, the monitor is reported as UP, even when if the monitor is DOWN. 

This is due to the `monitor.timespan` filter we previously had on this query. After discussion with design, the expectation is that the visualizations should ALWAYS correspond to the last check, even if the monitor has become off-schedule for any reason. 

Before

<img width="1257" alt="Screen Shot 2022-10-19 at 1 45 52 PM" src="https://user-images.githubusercontent.com/11356435/196772058-498c243f-9e74-408f-8638-1c225c1de61c.png">

After
<img width="1521" alt="Screen Shot 2022-10-17 at 1 42 22 PM" src="https://user-images.githubusercontent.com/11356435/196772056-c0a32811-0380-4f22-a069-0dae69cf18fd.png">

### Testing

1. Collect your local environment to the synthetics service in your preferred way
2. Create an always down monitor in Uptime or Synthetics app. Allow it to run a few times
3. Navigate to Synthetics app overview. You should see your monitor card in red reported as DOWN
4. Prevent your monitor from syncing by interrupting the connection between Kibana and the service, either by removing your tls certs or making the basic auth password incorrect
5. Wait for it to continue running about 15 minutes. 
6. Navigate to Synthetics App Overview. You should still see your monitor card reported in red as DOWN


### Recreating error in main 

1. Collect your local environment to the synthetics service in your preferred way
2. Create an always down monitor in Uptime or Synthetics app. Allow it to run a few times
3. Navigate to Synthetics app overview. You should see your monitor card in red reported as DOWN
4. Prevent your monitor from syncing by interrupting the connection between Kibana and the service, either by removing your tls certs or making the basic auth password incorrect
5. Wait for it to continue running about 15 minutes. 
6. Navigate to Synthetics App Overview. You should still see your monitor card reported in green as UP
